lib/iris/plot.py

```diff
@@ -271,7 +273,7 @@ def _invert_yaxis(v_coord, axes=None):
             axes.invert_yaxis()
 
 
-def _check_contiguity_and_bounds(coord, data, abs_tol=1e-4, transpose=False):
+def _check_contiguity_and_mask(coord, data, abs_tol=1e-4, transpose=False):
```

Comment:
I agree with the inclusion of `mask` but by not mentioning `bounds` it isn't clear that that's what it is referring to.
How about `_check_bounds_contiguity_and_mask`?

Reply:
Yeah that'll do.  It's a bit of a mouthful but it's private anyway so it shouldn't matter.